.travis.yml

```
@@ -15,6 +15,7 @@ env:
     - EXPORT_SERVICE_BULK_EXPORT_FREQUENCY="friday"
     - EXPORT_SERVICE_BULK_EXPORT_TIME="16:05"
     - FIRST_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME="16:15"
+    - EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME="23:00"
```

Unless there is a reason not to, we might as well go with what we'll actually be setting it to

```suggestion
    - EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME="00:05"
```